person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  def can_use_services?
+    of_age? || parent_permission
+  end
```

Comment:
- [ ] as the previous CR mentioned please add `@` to the `parent_permisision` to ensure we are using the instance variable declared, then it will look similar to below:
```
  def can_use_services?
    of_age? || @parent_permission
  end
```